istio/src/main/scala/io/buoyant/k8s/istio/IstioRequestAuthorizerFilter.scala

```diff
@@ -0,0 +1,40 @@
+package io.buoyant.k8s.istio
+
+import com.twitter.finagle.{Filter, Service}
+import com.twitter.util.{Duration, Stopwatch, Try}
+import io.buoyant.k8s.istio.mixer.MixerClient
+
+trait IstioRequestAuthorizerFilter[Req, Resp] extends Filter[Req, Resp, Req, Resp] {
+
+  def mixerClient: MixerClient
```

Comment:
TIOLI: this feels more like a dependency and less like something that needs to be implemented.  Could make this an abstract class instead of a trait and have MixerClient be a constructor argument.

Reply:
Absolutely agree, this is a leftover from the previous shape of this type. Let me fix it.

Reply:
Done!